ci-operator/config/openshift/release/openshift-release-master__ocp-4.6.yaml

```diff
@@ -31,6 +35,38 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: e2e-remote-libvirt-s390x
+  cron: 0 0 * * 1
+  steps:
+    cluster_profile: libvirt-s390x
+    env:
+      ARCH: s390x
+      MASTER_REPLICAS: "3"
+      NSS_GROUPNAME: ""
+      NSS_USERNAME: ""
+      NSS_WRAPPER_GROUP: ""
+      NSS_WRAPPER_PASSWD: ""
+      REMOTE_LIBVIRT_HOSTNAME: ocponz
+      REMOTE_LIBVIRT_HOSTNAME_1: ocponz2
+      REMOTE_LIBVIRT_URI: ""
+      WORKER_REPLICAS: "2"
+    workflow: ipi-libvirt
+- as: e2e-remote-libvirt-ppc64le
+  cron: 0 0 * * 1
+  steps:
+    cluster_profile: libvirt-ppc64le
+    env:
+      ARCH: ppc64le
+      MASTER_REPLICAS: "3"
+      NSS_GROUPNAME: ""
```

Comment:
I wonder if explicitly setting it here is overwritting the defaults.
With MASTER_REPLICAS, you set the default here as well. What if `""` is treated as though that is the value. Instead of undefined?